R/Package.R

```diff
@@ -0,0 +1,24 @@
+# @file ROhdsiWebApi.R
```

Comment:
What does this new file do?

Reply:
its modeled after https://github.com/OHDSI/SkeletonComparativeEffectStudy/blob/master/R/Package.R

it allows to provide meta information about the package. More importantly, it allows us to import packages/functions

Reply:
I'm not seeing this done in DatabaseConnector or SqlRender. @schuemie -- are we adding these Package.R files to packages now?

Reply:
@gowthamrao : I meant renaming the file, not just changing this line in the file.

@alondhe : Yes, all our R packages have such a file, for example [here](https://github.com/OHDSI/DatabaseConnector/blob/master/R/DatabaseConnector.R). But normally the file name is the same as the package name.

Reply:
Thanks. I made changes. Deleted package.R, modified ROhdsiWebApi.R